src/pages/Landing/index.js

```diff
@@ -1,10 +1,45 @@
 import React, { Component } from 'react';
 
+import Navbar from '../../components/Navbar';
+import Home from '../Home';
+import About from '../About';
+import Contact from '../Contact';
+
 class Landing extends Component {
+  state = {
+    siteName: 'T1CG Meetup Site',
+    currentPage: 'Home'
+  };
+
+  handlePageChange = page => {
+    this.setState({ currentPage: page });
+  };
+
+  renderPage = () => {
+    switch (this.state.currentPage) {
+      case 'Home':
+        return <Home />;
+        break;
```

Comment:
I don't know why I put a `break;` there even though I know that nothing gets executed after a `return`